server/routes/visitJourney/dateAndTime.ts

```diff
@@ -262,7 +262,8 @@ export default class DateAndTime {
     // if updating an existing booking, don't treat as an overbooking
     if (isUpdate) {
       const isOriginalSession =
-        originalVisitSession.date === selectedVisitSession.date && originalVisitSession.sessionTemplateReference
+        originalVisitSession.date === selectedVisitSession.date &&
+        originalVisitSession.sessionTemplateReference === selectedVisitSession.sessionTemplateReference
```

Comment:
Caught this mistake when adding the unit tests  - should have been checking template reference matches not just if original session present 🤦‍♂️ !